store-implementation/accumulo-store/src/main/java/uk/gov/gchq/gaffer/accumulostore/utils/AddUpdateTableIterator.java

```diff
@@ -54,7 +57,7 @@
     public static final String UPDATE_KEY = "update";
     public static final String REMOVE_KEY = "remove";
     public static final String ADD_KEY = "add";
-    private static final int NUM_REQUIRED_ARGS = 4;
+    private static final int NUM_REQUIRED_ARGS = 5;
```

Comment:
The pathToFileGraphLibrary is not a required argument. As you mention in the javadoc if it is not provided then NoGraphLibrary will be used instead. Therefore you should set the NUM_REQUIRED_ARGS to 4. Same for HBase.